db/migrations/mysql/000076_alter_terms_of_service_primary_key.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE PROCEDURE AlterIndex()
+BEGIN
+    DECLARE columnName varchar(26) default '';
+
+    SELECT IFNULL(GROUP_CONCAT(column_name ORDER BY seq_in_index), '') INTO columnName
+    FROM information_schema.statistics
+    WHERE table_schema = DATABASE()
+    AND table_name = 'UserTermsOfService'
+    AND index_name = 'PRIMARY'
+
+    IF columnName = 'UserId' THEN
```

Comment:
Like we do in the `down` script, can we check if the primary keys are not `(UserId, TermsOfServiceId)`?

Reply:
Sure. Do you know what is the expected value of columnName in that case? Is it `(UserId, TermsOfServiceId)`, or is it any other string?